src/main/java/org/sagebionetworks/bridge/services/StudyService.java

```diff
@@ -350,4 +355,15 @@ private Study phaseTransition(String appId, String studyId, StudyPhase targetPha
         
         return study;
     }
+    
+    private Set<String> getCustomEventIdsFromSchedule(String appId, String scheduleGuid) {
+        Schedule2 schedule = scheduleService.getSchedule(appId, scheduleGuid);
+        Set<String> existingEventIds = new HashSet<>();
+        if (schedule != null) {
+            existingEventIds = schedule.getSessions().stream()
+                    .flatMap(session -> session.getStartEventIds().stream())
+                    .filter(s -> s.startsWith("custom")).collect(Collectors.toSet());
```

Comment:
I would add the colon: s.startsWith("colon:")

Reply:
Added colon to "custom:"